emulsify.drush.inc

```diff
@@ -0,0 +1,299 @@
+<?php
+
+/**
+ * @file
+ * Contains Drush hooks. Inspired by Aeon drush commands.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function emulsify_drush_command() {
+  $items = array();
+
+  $items['emulsify'] = array(
+    'description' => 'Create an Emulsify theme.',
+    'arguments' => array(
+      'name'         => 'The name of your theme.',
+    ),
+    'options' => array(
+      'machine_name' => 'The machine-readable name of your theme. This will be auto-generated from the human-readable name if ommited.',
+      'description' => 'The description of your theme',
+      'path' => 'The destination of your theme. Defaults to "all" (themes).',
```

Comment:
I see you changed the code below, but this help text is inacurate. It should read something like
> The destination of your theme. Defaults to "/themes/custom". (we're not using "all" e.g. `/sites/all`)